src/components/genre-select.tsx

```diff
@@ -0,0 +1,37 @@
+import React, {useState} from "react";
```

Comment:
A common approach to naming react components is to use Pascal case, i.e. GenreSelect.tsx, SearchForm.tsx, etc. This is not a requirement, but you'll see this often in React projects, so I suggest that your rename them now, while there are still few components.